mkdocs_redirects/plugin.py

```diff
@@ -2,26 +2,51 @@
 Copyright 2019-2022 DataRobot, Inc. and its affiliates.
 All rights reserved.
 """
-import logging
+from __future__ import annotations
```

Comment:
note: We should probably drop Python 3.8 and upgrade requires-python to 3.9 in pyproject.toml, letting us (maybe) get rid of this import (though should be done in another PR).

Reply:
https://github.com/mkdocs/mkdocs-redirects/pull/74